diffs/filebeat2.diff

```diff
@@ -0,0 +1,4093 @@
+               tail_files
+@@ -1806,7 +1805,7 @@
+              </div>
+             </div>
+             <h5>
+-             <a id="_literal_symlinks_literal_3">
++             <a id="_symlinks_3">
+              </a>
+              <code class="literal">
+               symlinks
+@@ -1836,7 +1835,7 @@
+              Because this option may lead to data loss, it is disabled by default.
+             </p>
+             <h5>
+-             <a id="_literal_backoff_literal_3">
++             <a id="_backoff_3">
+              </a>
+              <code class="literal">
+               backoff
+@@ -1860,7 +1859,7 @@
+              value is reset to the initial value. The default is 1s.
+             </p>
+             <h5>
+-             <a id="_literal_max_backoff_literal_3">
++             <a id="_max_backoff_3">
+              </a>
+              <code class="literal">
+               max_backoff
+@@ -1908,7 +1907,7 @@
+              needs to be higher, it is recommended to close the file handler instead and let Filebeat pick up the file again.
+             </p>
+             <h5>
+-             <a id="_literal_backoff_factor_literal_3">
++             <a id="_backoff_factor_3">
+              </a>
+              <code class="literal">
+               backoff_factor
+@@ -1995,7 +1994,7 @@
+             The following configuration options are supported by all inputs.
+            </p>
+            <h5>
+-            <a id="_literal_enabled_literal_7">
++            <a id="_enabled_7">
+             </a>
+             <code class="literal">
+              enabled
+@@ -2012,7 +2011,7 @@
+             option to enable and disable inputs. By default, enabled is set to true.
+            </p>
+            <h5>
+-            <a id="_literal_tags_literal_7">
++            <a id="_tags_7">
+             </a>
+             <code class="literal">
+              tags
+@@ -2087,7 +2086,7 @@
+             sub-dictionary. If the custom field names conflict with other field names added by Filebeat, then the custom fields overwrite the other fields.
+            </p>
+            <h5>
+-            <a id="_literal_processors_literal_7">
++            <a id="_processors_7">
+             </a>
+             <code class="literal">
+              processors
+@@ -2109,7 +2108,7 @@
+             for information about specifying processors in your config.
+            </p>
+            <h5>
+-            <a id="_literal_pipeline_literal_7">
++            <a id="_pipeline_7">
+             </a>
+             <code class="literal">
+              pipeline
+html/en/beats/filebeat/master/filebeat-input-google-pubsub.html
+--- /dev/fd/63
++++ html/en/beats/filebeat/master/filebeat-input-google-pubsub.html
+@@ -230,7 +230,7 @@
+             described later.
+            </p>
+            <h4>
+-            <a id="_literal_project_id_literal">
++            <a id="_project_id">
+             </a>
+             <code class="literal">
+              project_id
```

Comment:
fix required

Reply:
fixed in https://github.com/elastic/beats/commit/a11ca121ee73c33767e4f2d2143debeaf52962ea